CHANGES.md

```diff
@@ -1,3 +1,23 @@
+## edge-19.7.4
+
+* CLI
+  * Made the `linkerd routes` command traffic-split aware; it now considers
```

Comment:
TIOLI: I would just remove everything from ";" afterwards since it feels too implementation-specific-y